src/util/kaldi-table-inl.h

```diff
@@ -340,6 +410,12 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
     // or the user called FreeCurrent().. note,
     // if when called by user we are in this state,
     // it means the user called FreeCurrent().
+    kObjectLoaded, // Called NextScpLine() and found the next    yes         yes
+    // object we need was alreay loaded in the last line.
```

Comment:
I think you are not accurately describing kObjectLoaded.  It implies that we have an object in the holder, corresponding to data_rxfilename_ (IIRC), but no extracted range.  This doesn't necessarily mean the object was from the last line, it could be that we just called LoadObject() and no range was specified.